apps/wing-console/console/server/src/router/test.ts

```diff
@@ -114,21 +115,41 @@ export const createTestRouter = () => {
       )
       .mutation(async ({ input, ctx }) => {
         await reloadSimulator(await ctx.testSimulator(), ctx.logger);
-        const response = await runTest(
-          await ctx.testSimulator(),
-          input.resourcePath,
-          ctx.logger,
-        );
+        try {
+          const response = await runTest(
+            await ctx.testSimulator(),
+            input.resourcePath,
+            ctx.logger,
+          );
 
-        const testsState = ctx.testsStateManager();
-        testsState.setTest({
-          id: input.resourcePath,
-          label: getTestName(input.resourcePath),
-          status: response.error ? "error" : "success",
-          time: response.time,
-        });
+          const testsState = ctx.testsStateManager();
+          testsState.setTest({
+            id: input.resourcePath,
+            label: getTestName(input.resourcePath),
+            status: response.error ? "error" : "success",
+            time: response.time,
+            datetime: Date.now(),
+          });
 
-        return response;
+          return response;
+        } catch (error: any) {
+          const testsState = ctx.testsStateManager();
+          testsState.setTest({
+            id: input.resourcePath,
+            label: getTestName(input.resourcePath),
+            status: "error",
+            time: 0,
+            datetime: Date.now(),
+          });
+          return {
+            response: "",
+            error: error.message,
```

Comment:
Can we include the full stack trace and not just the message?

Reply:
we have that already [here](https://github.com/winglang/wing/pull/6321/files#diff-a9fb0ffdeecbc0ac808e940483b7bb658a1a7f15a36d408f46d025de14309401R71)

```
bring cloud;

test "test" {
  throw "test error";
}
```

<img width="691" alt="image" src="https://github.com/winglang/wing/assets/5547636/f5fde512-fa4b-46df-a7c5-be2dc5ce7ebf">
 

I'm removing the try/catch that I added for debugging